client/state/selectors/is-sharing-buttons-save-successful.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Returns true if the save sharing buttons requests is successful
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Number}  siteId Site ID
+ * @return {Boolean}        Whether the request is successful or not
+ */
+export default function isSiteSettingsSaveSuccessful( state, siteId ) {
```

Comment:
Curious about this selector, and `isSavingSharingButtons` - will these be used in more than one place?  If not would it make sense to just have a generic `getSharingButtonsRequestStatus` selector?